### ISSUE TITLE
TODO/Ideas modifications/improvements of Dual 5uH LISN

### ISSUE THREAD

**Issue by bvernoux** [bvernoux/EMC_5uH_LISN#1]
Edited/Updated 10 December 2021:
1) Add the Simulation in the GitHub repository and check everything (to be checked in EEVBLOG forum) => Done in [V1_R0 ](https://github.com/bvernoux/EMC_5uH_LISN/releases/tag/V1_R0_12Dec2021) see directory [Simu](https://github.com/bvernoux/EMC_5uH_LISN/tree/master/Simu)
2) electronic_eel => Replace some 1N4148W by BAV99 to be checked => Done in [V1_R0 ](https://github.com/bvernoux/EMC_5uH_LISN/releases/tag/V1_R0_12Dec2021)
3) electronic_eel => Replace Keystone 575-4 with something better ? Replaced by KeyStone 575-8 [V1_R0 ](https://github.com/bvernoux/EMC_5uH_LISN/releases/tag/V1_R0_12Dec2021)
 3-1) At least allow to have 19mm space between Supply+/- 4mm BANANA JACK for BNC adapter... => Done in [V1_R0 ](https://github.com/bvernoux/EMC_5uH_LISN/releases/tag/V1_R0_12Dec2021)
4) electronic_eel => 4mm input connector with optional screw terminals would be nice
5) Switch to 4 Layers JLCPCB JLC7628 Stackup for good 50 Ohms Impedance ... => It seems not necessary especially as anyway it is limited to max 400MHz by Inductors
6) Check if S30-A75X => GDT 75V 5KA 2 POLE SURFACE MOUNT is required for 5uH DC-LISN => Yes Added (it is always good for safety of the hardware in extreme case)
7) Check the BOM to fix/optimize it (it can be generated with https://github.com/hildogjr/KiCost) => Work in progress waiting PCB for [V1_R0 ](https://github.com/bvernoux/EMC_5uH_LISN/releases/tag/V1_R0_12Dec2021) to push the new optimized BOM from JLCPCB
8) HardwareBoffin (on Twitter) => Use open air inductances to reach higher frequency (especially for 10KHz to 200MHz+)
    - Today High Current Inductor are 5 x 1.1uH (per LISN) Würth WE-HCI 744314110 (see KiCad Schematic BOM) but they are pretty expensive and it is interesting if we can find other alternative cheaper and or with better performance
    - The aim is to find some ready to buy & solder SMT part to avoid user's to build their own Inductor when possible ...
9) TBD ...

**Comment by wm93** [comment]
Regarding 8. you might consider the following inductors?
- WE 7447720010
(1uH, Fres=195MHz, 6.9Adc 15Asat, THT, 10pc: 0,644 € each) ->[Mouser](https://www.mouser.de/ProductDetail/Wurth-Elektronik/7447720010?qs=sPbYRqrBIVnpxnA1X%252BwmqA%3D%3D)

- Bourns RLB9012-1R0ML
(1uH, Fres=150MHz, 10Adc, THT, 10pc: 0,561€ each) -> [Mouser](https://www.mouser.de/ProductDetail/Bourns/RLB9012-1R0ML?qs=dB0LedT%252BmmABces9278Nbg%3D%3D)

- Coilcraft SER2009-102MLD (more expensive but much higher dc current + self-resonance)
(1uH, Fres=488MHz, 54Adc, 29Asat, SMD, 10pc: 2,31€ each) -> [Mouser](https://www.mouser.de/ProductDetail/Coilcraft/SER2009-102MLD?qs=zCSbvcPd3pab%252BkLHUUwP3w%3D%3D)

**Comment by bvernoux** [comment]
Thanks for the feedback and references
My remarks:
- Würth WE 7447720010 is similar in size to Würth WE-HCI 744314110 but will requires PCB change as it is through hole
  - Potentially we can do 1 footprint to use WE 7447720010 or Würth WE-HCI 744314110
  - The biggest advantage is the price is >4.5x lower vs Würth WE-HCI 744314110 (I will buy few to test them)
- Bourns RLB9012-1R0ML is similar in size (9mm x 9mm x 12.2mm) to Würth WE-HCI 744314110 but will requires PCB change as it is through hole
  - Potentially we can do 1 footprint to use Würth WE 7447720010 or Würth WE-HCI 744314110 or Bourns RLB9012-1R0ML
  - The biggest advantage is the price is >4x lower vs Würth WE-HCI 744314110 (I will buy few to test them)
- Coilcraft SER2009-102MLD is very big (18.7mm x 19.18mm x 8.64mm vs Würth WE-HCI 744314110=7mm x 6.9mm x 4.8mm) and will requires change of the PCB (I cannot test such big Inductor on actual PCB as maximum we can place 12mm Inductors)
  - It is very interesting if user's need to increase the max power which is limited today to 5A (but it will requires also lot of other modifications and so far I do not plan to increase the power)

For information:
I have launched a production of 5 boards with a new version (called V1 R0) of actual design (that I will push on github soon) including following improvements:
- Added simulation with LTSpice of the 10dB Attenuator / Transient Filter
- Replaced some 1N4148W by BAV99 from Nexperia (to reach up to 400MHz flat 10dB Attenuator / Transient Filter)
- Changed space to have 19.06mm between Supply / DUT "+" <=> "GND" <=> "-" => 4mm BANANA JACK for BNC adapter
- Changed Keystone Electronics 575-4 which are too short for banana by Keystone Electronics 575-8

**Comment by wm93** [comment]
Hey, thanks for your quick reply :)
I did also some simulations and overlayed them in a graph with the CISPR16/CISPR25 given Impedanze/Phase graphs for the 5uH ANs.

Since you started the project, where would you like me to post that information (if at all)? Do you want to work here in the repo or should I post those discussions in the EEVblog thread you mentioned in the readme?

**Comment by bvernoux** [comment]
I have just pushed the version [V1_R0 ](https://github.com/bvernoux/EMC_5uH_LISN/releases/tag/V1_R0_12Dec2021) (I have updated the 1st comment too)
* I think sharing file in EEVBlog forum is a very bad idea as there is no source control and it is a real mess it is why I have created this GitHub to have everything up to date at one place with source control.
* You can do a Pull Request and add your simulation in Simu directory (please use LTSpice and or only open source/free tool for simulation)

**Comment by wm93** [comment]
Should we do some proper folder structure? E.g.

master
- readme.md
- KiCAD
- Simulation
  - 10dBAttenuator_TransientLimiter
    - 10dBAttenuator_TransientLimiter.asc
    - 10dBAttenuator_TransientLimiter_BAV99.asc
    - BAV99.mod
  - CISPR16_5uH_LISN
  - CISPR25_5uH_LISN
  - ...
- Documentation
 
For example, I did a simulation and plotted it together with the CISPR given impedances/phases of the 5uH LISN. I would like to document that somewhere.
![impedance_vs_frequency](https://user-images.githubusercontent.com/11892467/145673217-28f45b74-bf17-4db3-a392-1b471e3e6e7a.png)
![phase_vs_frequency](https://user-images.githubusercontent.com/11892467/145673512-430d2c01-7dea-41dc-a3f4-f14ff440a65d.png)

**Comment by bvernoux** [comment]
Yes if there is lot of simulation it is better to have a folder structure like in your proposal (so far I have done only 10dBAttenuator_TransientLimiter but other simulation are welcome)

**Comment by wm93** [comment]
Do I need to get access rights to the repository, before I can do changes? Tried to "git mv Simu Simulation; git commit; git push" but it wont let me do it.

**Comment by bvernoux** [comment]
If you want to contribute you shall work on your own fork of this repository, then do a Pull Request to this one and I will check everything ... and merge it if all is ok

**Comment by Brianwongjie** [comment]
Hello, I found your Github website from the eevblog, but I couldn't find the original design information about 50uh Lisn in the entire forum. I would like to inquire if there are any BOMs or Gerber related to saving 50uh Lisn here

**Comment by bvernoux** [comment]
> Hello, I found your Github website from the eevblog, but I couldn't find the original design information about 50uh Lisn in the entire forum. I would like to inquire if there are any BOMs or Gerber related to saving 50uh Lisn here

The purpose of this Repository is to only work on EMC 5uH LISN (low voltage) and it is not planned to add the 50uH LISN as I do not have usage of such higher voltage LISN which can be lethal (see https://www.quora.com/What-is-the-safe-limit-of-DC-voltage-for-humans-to-touch) if not used correctly.
